server/memphis_handlers_schemas.go

```diff
@@ -431,31 +441,28 @@ func (sh SchemasHandler) RemoveSchema(c *gin.Context) {
 	if !ok {
 		return
 	}
-	for _, name := range body.SchemasName {
+	for _, name := range body.SchemaNames {
 		schemaName := strings.ToLower(name)
 		exist, schema, err := IsSchemaExist(schemaName)
 		if err != nil {
 			serv.Errorf("RemoveSchema error: " + err.Error())
 			c.AbortWithStatusJSON(500, gin.H{"message": "Server error"})
 			return
 		}
-		if !exist {
-			serv.Warnf("Schema does not exist")
-			c.AbortWithStatusJSON(configuration.SHOWABLE_ERROR_STATUS_CODE, gin.H{"message": "Schema does not exist"})
-			return
-		}
-		DeleteTagsBySchema(schema.ID)
+		if exist {
+			DeleteTagsBySchema(schema.ID)
 
+		}
 	}
-	err := sh.findAndDeleteSchema(body.SchemasName)
+	err := sh.findAndDeleteSchema(body.SchemaNames)
```

Comment:
instead doing this do the following, in the exist block insert the schema Ids into an array of schema Ids, here instead calling this function with names call it with Ids array, inside the delete function you will not have to query the DB again, this way you will be able to remove the schema and the schema versions more efficiently